students/vkis/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,57 @@
+# Task 1
+print("\nTask 1")
+tuple1 = (2, 123.4567, 10000, 12345.67)
+
+file_num = tuple1[0]
+if file_num >= 10:
+	file_name = "file0" + str(file_num)
+else:
+	file_name = "file00" + str(file_num)
+
+print("{}: {:.2f}, {:.2E}, {:.3E}".format(file_name,tuple1[1],tuple1[2],tuple1[3]))
+
+
+# Task 2
+print("\nTask 2")
+print(f"{file_name}: {tuple1[1]:.2f}, {tuple1[2]:.2E}, {tuple1[3]:.3E}")
```

Comment:
I know this is new in 3.6 `print(f"` but you might want to hold off using this, as no other python version supports it.